src/main/features/response/tasks/howMuchDoYouOweTask.ts

```diff
@@ -0,0 +1,18 @@
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Validator } from 'class-validator'
+import { HowMuchDoYouOwe } from 'response/form/models/howMuchDoYouOwe'
+import { YesNoOption } from 'models/yesNoOption'
+
+const validator = new Validator()
+
+export class HowMuchDoYouOweTask {
+  static isCompleted (responseDraft: ResponseDraft): boolean {
```

Comment:
please correct me if I'm wrong, but this class is not unit tested. 

REQUIRE: please add unit tests for `isCompleted`

Reply:
Done 👍